scripts/transition_metrics.py

```diff
@@ -43,6 +72,7 @@ def rx_commission_decommission(filename, non_lwr):
         names of nonreactor facilities in the simulation
```

Comment:
Here you say "names of nonreactor facilities in the simulation", but in `prototype_totals`, the equivalent parameter is described as "names of prototypes that are not LWRs". Pick whichever one is more correct to both of them for consistency. Consider changing to `non_reactor_names` or something similar if the parameter should be for all nonreactors and not just non-LWRs.